src/Common/tests/System/Net/Configuration.Certificates.cs

```diff
@@ -19,7 +19,7 @@ public static partial class Certificates
             private const string CertificatePassword = "testcertificate";
             private const string TestDataFolder = "TestData";
             private static Mutex m = new Mutex(false, "Global\\CoreFXTest.Configuration.Certificates.LoadPfxCertificate");
-            private const int MutexTimeout = 5 * 1000;
+            private const int MutexTimeout = 30 * 1000;
```

Comment:
nit:  a comment at the end of this line like "// 30 seconds" would be useful.  It isn't obvious that the units are milliseconds.